packages/models/src/token-generation-readmodel/dynamoDB-keys.ts

```diff
@@ -0,0 +1,120 @@
+import { z } from "zod";
+import {
+  EServiceId,
+  DescriptorId,
+  AgreementId,
+  PurposeId,
+  ClientId,
+  TenantId,
+} from "../brandedIds.js";
+
+export const PlatformStatesEServiceDescriptorPK = z
+  .string()
+  .brand(`ESERVICEDESCRIPTOR#eServiceId#descriptorId`);
+export type PlatformStatesEServiceDescriptorPK = z.infer<
+  typeof PlatformStatesEServiceDescriptorPK
+>;
+export const makePlatformStatesEServiceDescriptorPK = ({
+  eserviceId,
+  descriptorId,
+}: {
+  eserviceId: EServiceId;
+  descriptorId: DescriptorId;
+}): PlatformStatesEServiceDescriptorPK =>
+  `ESERVICEDESCRIPTOR#${eserviceId}#${descriptorId}` as PlatformStatesEServiceDescriptorPK;
```

Comment:
Couldn't we use `unsafeBrandId` in these? You should also move all the branded types to `models/src/brandedIds.ts`

Reply:
Updated